fastlane/lanes/screenshots.rb

```diff
@@ -2,6 +2,19 @@
 
 require 'fileutils'
 
+# Be sure to keep the `fastlane/screenshots.json` and `fastlane/jetpack_screenshots.json` files in sync with that list as well
+#
+# Screen Sizes Reference Charts:
+# - https://size-charts.com/topics/screen-size-charts/apple-iphone-size/
+# - https://size-charts.com/topics/screen-size-charts/apple-ipad-size-chart/
+#
+SCREENSHOT_SIMULATORS = [
+  'iPhone Xs Max', # 6.5in - 2688x1242 @ 458 ppi
+  'iPhone 8 Plus', # 5.5in - 1920x1080 @ 401 ppi -- !!! FIXME: `canvas_size` in `fastlane/screenshots.json` does not match ([1242, 2208])
```

Comment:
I've decided not to address this in this PR, because:
 - Fixing that would require updating a lot of coordinates (offsets and sizes) in `screenshots.json`, and potentially to also update the device frame PNG used for that device… and to do that properly we need to have the Screenshots UI Tests (producing the raw screenshots) working first, to then test how the promo screenshots using that `screenshots.json` would need to be adjusting… so this will definitively be for a later PR
 - Also, the aspect ratio of 2208/1242 = 1920/1080 = 16/9 is the same for both models… so maybe that does not matter that much after all (i.e. the raw screenshots, generated from an iPhone 8 Plus with dimensions 1920x1080, will _not_ be distorted and will keep a coherent aspect ratio when inserted into the frame made for iPhone 7 Plus in the promo screenshot)

Still, it would be nice to keep that consistent next time we make the screenshots UI Tests work again (hopefully soon, see pbArwn-4Pq-p2#comment-6053) and then potentially update the list of devices and the `screenshots.json` files with more modern devices (but we need to have the UI Test and raw screenshots work first as they are a necessary input for iterating on the promo screenshots)